js/main.js

```diff
@@ -143,6 +143,59 @@ $('body').on('keypress', 'input', function(event) {
   }
 });
 
+//keyboard shortucts
+window.keyShortcuts = {
+	discover: 		'd',
+	myPage: 		'h',
+	customizePage: 	'e',
+	create: 		'n',
+	purchases: 		't',
+	sales: 			's',
+	cases: 			'b',
+	settings: 		','
```

Comment:
Ah. Missed that. Can it be something easier to see? It's not very visible in the menu.